hypervisor/qemu/qemu.go

```diff
@@ -354,4 +373,68 @@ func (qc *QemuContext) arguments(ctx *hypervisor.VmContext) []string {
 		"-fsdev", fmt.Sprintf("local,id=virtio9p,path=%s,security_model=none", ctx.ShareDir),
 		"-device", fmt.Sprintf("virtio-9p-pci,fsdev=virtio9p,mount_tag=%s", hypervisor.ShareDirTag),
 	)
+
+	//lazy add nic device
+	for i, info := range qc.interfaces {
+		params = append(params,
+		"-netdev", fmt.Sprintf("tap,fd=%d,id=%s", info.fd, info.deviceName),
```

Comment:
Yes，you're right, in this branch I forget something and didn't test the nic after the qemu launch.In order to fix this , we should modify the code under path /runv/lib/utils/daemon.go:
	//Close all open file descriptors
//	for (i = 0; i < num; i++) {
//		if (fds[i] == 0 || fds[i] == 1 || fds[i] == 2)
//			continue;
//		printf("\n\nI didnt close any file descriptor\n\n");
//		close(fds[i]);
//	}
just don't close the fds.If want to optimize , we can just leave the nic fd open.